garage.com/src/cssXpath/cssselectorproject.txt

```diff
@@ -0,0 +1,31 @@
+1.Page:/text-box. Write a selector that will return the ‘Full Name’ textboxusing the ID attribute. Hint: This element has the ‘input’ tag.
+
+input[id='userName']
+
+2.Page: /buttons. Write a selector that will return all 3 buttons shown below.
+
+button[id='doubleClickBtn']
+button[id='rightClickBtn']
+button[id='L2NbR']
+another way 
+button[type*='button'][class='btn btn-primary']

```

Comment:
Tag name only works:
button

Reply:
we use each of them seprate here like button[id=['rightClickBtn'] for each button with seprate id ?